### PR TITLE
Test book

### PR DESCRIPTION
**Pull request by DenysMaksymKachalup**

### CODE REVIEW

**Comment by DenysMaksymKachalup** [comment]
image
image

**Comment by DenysMaksymKachalup** [comment]
image

File: src/test/java/com/example/springonlinebookstore/controller/BookControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        MvcResult result = mockMvc.perform(post("/books")
+                        .content(jsonRequest)
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().is(200))
+                .andReturn();
+
+        BookDto actual = objectMapper.readValue(
+                result.getResponse().getContentAsString(), BookDto.class);
+        assertNotNull(actual);
+        EqualsBuilder.reflectionEquals(getBookDto(), actual, "id");
```
Comment (sokimaaa) [inline src/test/java/com/example/springonlinebookstore/controller/BookControllerTest.java]: use `jsonPath(...)` to simplify testing output of the controller. 
```suggestion
.andExpect(jsonPath("$[0].foo", is("bar"))) // example
```

Comment (sokimaaa) [reply src/test/java/com/example/springonlinebookstore/controller/BookControllerTest.java]: refactor all controller tests regarding this comment

**Pull request merged**